CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ Steps:
 2. Create a ``feature`` branch to hold your development changes:
 
    ```bash
-   $ git checkout -b my-feature
+   $ git checkout -b {feature|issue}/feature-name
```

Comment:
Wasn't it going to be feature/issue/feature-name?

Reply:
yes